.env

```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=ROHIT
```

Comment:
For security reasons, `env` file is generally not commited to git. You can easily add an environment variable on your hosting site for this.